File: DictionaryProject/WordBuilder/WeightedCommand.vb
Public Class WeightedCommand
    Private _Command As CommandBase
    Public Property Command() As CommandBase
        Get
            Return _Command
        End Get
        Set(ByVal value As CommandBase)
            _Command = value
        End Set
    End Property

    Private _Weight As Double
    Public Property Weight() As Double
        Get
            Return _Weight
        End Get
        Set(ByVal value As Double)
            _Weight = value
        End Set
    End Property
End Class

File: monowordbuilder/wordbuilderbase/Project/WeightedCommand.cs
using System;
using Whee.WordBuilder.Model.Commands;

namespace Whee.WordBuilder.Model
{
	public class WeightedCommand
	{
		private CommandBase _Command;
		public CommandBase Command {
			get { return _Command; }
			set { _Command = value; }
	    }
	    
		private double _Weight;
		public double Weight {
			get { return _Weight; }
			set { _Weight = value; }
	    }
	}
}